scss/base/_variables.scss

```diff
@@ -134,7 +134,7 @@ $zindex-fullscreen:                  2147483647 !default; //in screenfull, here
 $zindex-state-screen:                      1000 !default;
 $zindex-vr-controls:                       1001 !default;
 $zindex-up-next-panel:                     1002 !default;
-$zindex-unmute:                           13000 !default;
+$zindex-unmute:                      2147483647 !default;
```

Comment:
Is the button inside a container with a lower z-index? If not then I think this large value might make it hard for customers to have elements that are on top of the player, like navigation bars, etc.

Reply:
It's inside the playing screen and ad screen containers, which don't appear to have defined z indices. I've had some IMA ads have click priority over the mute icon, so we're in a tough spot to decide what this z-index should be. I can change this to 19999 to be right under our "SUPER_Z_INDEX" for the time being.

Reply:
Oh ok I see, sounds good.